core/invoker/src/main/resources/application.conf

```diff
@@ -155,6 +155,8 @@ whisk {
       #aka 'How long should a container sit idle until we kill it?'
       idle-container = 10 minutes
       pause-grace = 50 milliseconds
+      lease-grace = 5 seconds
```

Comment:
seems not used?

Reply:
Great found, doesn't use forever. 
So i deleted it as well.